### DISCUSSION TITLE
Testing branch now active --

### DISCUSSION THREAD

**Discussion by iam-sysop** [thecarnie/cycle2#3] [📣Announcements]
I have added a [Testing](https://github.com/thecarnie/cycle2/tree/Testing) branch to the repository.

This branch is **NOT RECOMMENDED for PRODUCTION USE** 

It may contain breaking changes to Cycle2 as feature changes are implemented, deprecated features updated, and so forth.  

On the other hand, it will always contain the most RECENT version of the code I am working with, as well as a "built" copy of the Cycle2 plugin in the ./build folder for your own testing.

**Comment by webimagedesigns** [comment]
Hi @thecarnie I had reported the jQuery Migrate issue to you - see below
I have legacy sites and I was hoping for an updated version that fixed the deprecated code and did not introduce any new features to keep those working. Would that be possible? I would be very grateful if it was.

----issues ---
jQuery.type and jQuery.isFunction() are being flagged as deprecated by jQuery Migrate in WordPress 5.6 for jquery.cycle2.min.js. I am using jQuery.cycle2.min.js
/*!

jQuery Cycle2; version: 2.1.6b build: 20200917
http://jquery.malsup.com/cycle2/
updated repo at https://github.com/thecarnie/cycle2/
Copyright (c) 2020 M. Alsup; Dual licensed: MIT/GPL
*/

**Reply by iam-sysop** [reply]
Please check here for a test version - if you are willing to try the test branch - https://github.com/thecarnie/cycle2/tree/Testing/build

In that folder, you can pull down copies of the jquery.cycle2.min.js and jquery.cycle2.min.js.map files -- I have tested them against a dozen sliders I have on different websites and am not catching any errors - but I am hesitant to roll them into the build until it's tested more.

Simply rename the current copies to something like jqeuery.cycle2.min.js.production - then upload the test build, and perform a hard-refresh of the site in the browser.  If your cycles fail to load, delete the freshly uploaded copies, and rename the old ones back -

**Comment by webimagedesigns** [comment]
I have put your new code on two of my legacy systems and all appears to be
working fine, AND the jQuery migrate issues that were flagged by WordPress
are gone. Thank you so much for the updated code!!




On Fri, Jan 29, 2021 at 3:50 PM sysop <notifications@github.com> wrote:

> Please check here for a test version - if you are willing to try the test
> branch - https://github.com/thecarnie/cycle2/tree/Testing/build
>
> In that folder, you can pull down copies of the jquery.cycle2.min.js and
> jquery.cycle2.min.js.map files -- I have tested them against a dozen
> sliders I have on different websites and am not catching any errors - but I
> am hesitant to roll them into the build until it's tested more.
>
> Simply rename the current copies to something like
> jqeuery.cycle2.min.js.production - then upload the test build, and perform
> a hard-refresh of the site in the browser. If your cycles fail to load,
> delete the freshly uploaded copies, and rename the old ones back -
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/thecarnie/cycle2/discussions/3#discussioncomment-322054>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGFHBUFC4GEVKE7XCGMDTL3S4MNSBANCNFSM4WSO5HWQ>
> .
>


-- 
*Happy with my service? Please give me a review on google
<https://www.google.com/search?q=web+image+designs&oq=web+image+designs&aqs=chrome..69i57j69i60l3j69i65j0.3615j0j7&sourceid=chrome&ie=UTF-8#lrd=0x89e409112e7b62b5:0x13e88b2507e5ab14,1,,,>.
If you are unhappy about anything, please let me know so I can fix it.*

*Judy Vedder*
President
Direct Dial: 774-275-0391 <+1774-275-0391>



WebImageDesigns.com <https://webimagedesigns.com/>

**Reply by iam-sysop** [reply]
I have packaged up the cycle2 into a new release available on the main page or here on the releases page - there is no change from what you have installed already - the code was merged direct from the test branch.

https://github.com/thecarnie/cycle2/releases/2.1.6c

**Comment by webimagedesigns** [comment]
Thank you so much!

On Tue, Feb 2, 2021 at 2:15 PM sysop <notifications@github.com> wrote:

> I have packaged up the cycle2 into a new release available on the main
> page or here on the releases page - there is no change from what you have
> installed already - the code was merged direct from the test branch.
>
> https://github.com/thecarnie/cycle2/releases/2.1.6c
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/thecarnie/cycle2/discussions/3#discussioncomment-331985>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGFHBUGHCD54ZTQPA5C372TS5BFMRANCNFSM4WSO5HWQ>
> .
>


-- 
*Happy with my service? Please give me a review on google
<https://www.google.com/search?q=web+image+designs&oq=web+image+designs&aqs=chrome..69i57j69i60l3j69i65j0.3615j0j7&sourceid=chrome&ie=UTF-8#lrd=0x89e409112e7b62b5:0x13e88b2507e5ab14,1,,,>.
If you are unhappy about anything, please let me know so I can fix it.*

*Judy Vedder*
President
Direct Dial: 774-275-0391 <+1774-275-0391>



WebImageDesigns.com <https://webimagedesigns.com/>
